shared/scss/components/ckan/dataset-list.scss

```diff
@@ -94,4 +94,8 @@
     display: inline-block;
     padding: $padding-small-vertical $padding-small-horizontal;
   }
-}
\ No newline at end of file
+
+  a {
+    color: $label-text-color;
+  }
+}
```

Comment:
When I switched the places of `a` and `li` elements in `package_item.html` above, the `a` color overrode the `label` color so I added the above color styling for `.dataset-resources` class.